db/seeds.rb

```diff
@@ -34,44 +79,38 @@
 #---------------------------------OWNER-------------------------------------#
 #############################################################################
 
+# Prevents Gary from having other plusle and minun pokemons for demo
```

Comment:
This is for the demo presentation, I don't want duplicate 'plusle' and 'minun' pokemons.